dmt/settings_shared.py

```diff
@@ -26,7 +26,6 @@
 ]
 
 TEMPLATE_CONTEXT_PROCESSORS += [  # noqa
-    'django.contrib.messages.context_processors.messages',
```

Comment:
the messages stuff is already included in ccnmtlsettings, so it's not necessary to have it in dmt's settings